doc/api/stream.md

```diff
@@ -325,6 +349,9 @@ implementations that implement the `writable.\_writev()` method can perform
 buffered writes in a more optimized manner.
 
 ##### writable.end([chunk][, encoding][, callback])
+<!-- YAML
+added: v0.9.9
+-->
```

Comment:
@italoacasas I’d still prefer to go with the version it was added in; adding some sort of changelog for individual methods is definitely planned.